src/commands/runtime/action/create.js

```diff
@@ -146,8 +187,7 @@ ActionCreate.flags = {
     description: 'the maximum log size LIMIT in MB for the action (default 10)' // help description for flag
```

Comment:
@purplecabbage I think this should not reference the default value - which comes from the server config, but is not exposed afaik.